### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by bobynator93** [mampfes/hacs_waste_collection_schedule#1796] [Ideas]
I would like to see a translation so that "... in X days", for example, is translated as "in X Tagen" in German.

**Comment by 5ila5** [comment]
I don't know "where in X days" is hard-coded anywhere. Are you talking about the sensor? You can simply change the value_tamplate of the sensor. If this does not help, please be more specific where you found that text.

**Reply by bobynator93** [reply]
oh okay thanks, I will try.
I think its this line:
https://github.com/mampfes/hacs_waste_collection_schedule/blob/e89b1251bc93212660886475f9c639805533135f/custom_components/waste_collection_schedule/sensor.py#L183

**Reply by 5ila5** [reply]
This is the default case if no value template is provided, simply add `value_template: "in {{ value.daysTo }} Tagen"` to your sensor config. I don't think there is a need to translate this default case.
